test/functional/blockchain.py

```diff
@@ -6,6 +6,7 @@
 
 Test the following RPCs:
     - gettxoutsetinfo
+    - getdifficulty
     - verifychain
```

Comment:
Might as well fix this docstring while you're at it. The following RPCs are also tested:

- getbestblockhash
- getblockhash
- getblockheader